[2023-07-16T03:32:38.273+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.install_libraries manual__2023-07-16T03:32:37.587120+00:00 [queued]>
[2023-07-16T03:32:38.283+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.install_libraries manual__2023-07-16T03:32:37.587120+00:00 [queued]>
[2023-07-16T03:32:38.283+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-16T03:32:38.300+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): install_libraries> on 2023-07-16 03:32:37.587120+00:00
[2023-07-16T03:32:38.305+0000] {standard_task_runner.py:57} INFO - Started process 675 to run task
[2023-07-16T03:32:38.307+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'install_libraries', 'manual__2023-07-16T03:32:37.587120+00:00', '--job-id', '441', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmphz0h6m0k']
[2023-07-16T03:32:38.310+0000] {standard_task_runner.py:85} INFO - Job 441: Subtask install_libraries
[2023-07-16T03:32:38.357+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.install_libraries manual__2023-07-16T03:32:37.587120+00:00 [running]> on host 75cbd5457fb2
[2023-07-16T03:32:38.447+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='install_libraries' AIRFLOW_CTX_EXECUTION_DATE='2023-07-16T03:32:37.587120+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-16T03:32:37.587120+00:00'
[2023-07-16T03:32:38.447+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-16T03:32:38.448+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install networkx']
[2023-07-16T03:32:38.459+0000] {subprocess.py:86} INFO - Output:
[2023-07-16T03:32:39.116+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-07-16T03:32:39.255+0000] {subprocess.py:93} INFO - Requirement already satisfied: networkx in /home/***/.local/lib/python3.7/site-packages (2.6.3)
[2023-07-16T03:32:43.993+0000] {subprocess.py:93} INFO - 
[2023-07-16T03:32:43.994+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.1.2 -> 23.2
[2023-07-16T03:32:43.994+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-07-16T03:32:44.154+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-16T03:32:44.182+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=install_libraries, execution_date=20230716T033237, start_date=20230716T033238, end_date=20230716T033244
[2023-07-16T03:32:44.224+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-16T03:32:44.245+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
